app/controllers/chat_channels_controller.rb

```diff
@@ -64,20 +64,19 @@ def unfollow
   def create
     params.require([:type, :id, :name])
     guardian.ensure_can_create_chat_channel!
-    raise Discourse::InvalidParameters unless ["topic", "category"].include?(params[:type].downcase)
+    raise Discourse::InvalidParameters unless ["category"].include?(params[:type].downcase)
     raise Discourse::InvalidParameters.new(:name) if params[:name].length > SiteSetting.max_topic_title_length
 
-    creating_topic_channel = params[:type].downcase === "topic"
-    chatable_type = creating_topic_channel ? "Topic" : "Category"
+    chatable_type = "Category"
     existing_args = {
       chatable_type: chatable_type,
       chatable_id: params[:id]
     }
-    existing_args[:name] = params[:name] unless creating_topic_channel
+    existing_args[:name] = params[:name]
     exists = ChatChannel.exists?(existing_args)
 
     if exists
-      translation_key = creating_topic_channel ? "channel_exists_for_topic" : "channel_exists_for_category"
```

Comment:
Can the `channel_exists_for_topic` translation be removed from the locale file or is it used somewhere else?

Reply:
I think we can remove it yes